### ISSUE TITLE
Enable support for the Insights API 

### ISSUE THREAD

**Issue by rtwlabs** [WebDevStudios/wp-search-with-algolia#268]
Is there the potential to enable the Algolia insights API in the plugin to allow sending back user activity so the AI dynamic re-ranking, conversions tracking and click events can be recorded and used as well?

https://www.algolia.com/doc/guides/algolia-ai/re-ranking/?utm_medium=page_link&utm_source=dashboard#set-up-dynamic-re-ranking

Or is something like that beyond the scope of what you are aiming to do with this plugin at this point in time?

**Comment by tw2113** [comment]
Hi @rtwlabs at the moment, out of box, I don't believe we have things in the optimal position for this out of box, but that doesn't mean it can't be done as a whole since all of the templates and Autocomplete/InstantSearch library usage are customizable to meet each individual website's goals.

I do know that we're working on getting things set up better to help with these types of topics with hopefully as little forced coding as possible, or at least get it to a point where it's more UI configured.

**Comment by rtwlabs** [comment]
@tw2113 good to know and thanks for the update. I might experiment a little with our integration in the meantime. Didn't want to devote time to something that might not be in the realm of possibility.

**Comment by tw2113** [comment]
@rtwlabs for what it's worth, the Algolia documentation is quite robust and full of information, even if it sometimes takes a little bit to mentally parse out and find what you need.

https://www.algolia.com/doc/

I use them all the time lately.

**Comment by rtwlabs** [comment]
Yes I've found it to be fairly useful. Been working with our account rep with them recently as we reviewed our usage. And they flagged that the plugin did not have the insights data to take advantage of all the features we could have. Hence my query in the first place.

**Comment by tw2113** [comment]
yep yep, but we hope to get there. We're definitely active with the plugin.

**Comment by rtwlabs** [comment]
Oh, for sure, and it's excellent that you've picked it up after Algolia let their plugin go. Some of the recent changes to reduce jquery etc., have been great.

**Comment by tw2113** [comment]
It's presently commented out by default, but we have example code for the Insights API present and ready to use in the `instantsearch-modern.php` as part of version 2.9.0.

Closing as part of that fact.
